app/assets/stylesheets/application.css.scss

```diff
@@ -39,3 +39,9 @@
     border: solid 1px $dark-red;
   }
 }
+
+.pagination {
+  text-align: center;
+  background-color: white;
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.